src/default-config.ts

```diff
@@ -6,3 +6,7 @@ export const DEFAULT_PERPLEXITY_MODEL = 'sonar-medium-chat';
 
 export const DEFAULT_SYSTEM_PROMPT =
   'You are a helpful assistant responding in a concise manner to user questions.';
+
+export const DEFAULT_FILE_PROMPT = `Below is a file attached to conversation.
```

Comment:
Could we simplify it? e.g. "Answer using following content as a reference. If you can't finde the answer to the question in provided content just respond "I don't know".

CONTENT: {fileCOntent}"

Reply:
Done